### ISSUE TITLE
Spanish translation: Booster updates from 1/11/23 on 2 covid pages

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#51918]
## Issue Description

We made a few updates to 2 COVID-19 benefit detail pages that we need translated in Spanish.

On the [main vaccines page](https://www.va.gov/health-care/covid-19-vaccine/) we made updates to the blue eligibility box.

On the [boosters](https://www.va.gov/health-care/covid-19-vaccine/booster-shots-and-additional-doses/) page we made the same changes in the "When to get your updated booster shot section."  This wasn't sent explicitly to the translators since it's the same same sentences on the first page. 

We need to update the Spanish translation pages with these edits. 

Word doc sent to translators: [https://app.zenhub.com/files/133843125/2fc17198-1929-4f0a-8d25-b14f2725c9f7/download](https://app.zenhub.com/files/133843125/2fc17198-1929-4f0a-8d25-b14f2725c9f7/download)

## Tasks
- [x] Update main vaccine page (Aliyah)
- [x] Update booster page (Aliyah)
- [x] Copyedit main vaccine page (Megan)
- [x] Copyedit booster page (Megan)
- [x] Publish main vaccine page (Megan)
- [x] Publish booster page (Megan)

## Acceptance Criteria
- [x] Spanish page is live with updates

**Comment by RLHecht** [comment]
@aliyahblackmore Here's the Spanish translation back: [https://app.zenhub.com/files/133843125/acf1a7d5-6ce7-422f-9eef-a7fc28fd6acb/download](https://app.zenhub.com/files/133843125/acf1a7d5-6ce7-422f-9eef-a7fc28fd6acb/download)

**Comment by aliyahblackmore** [comment]
Thanks Randi! 

@megzehn this is ready for you! We're only updating the content that we got back from the translators in this document, so there will be some difference in the remainder of the live English pages and the Spanish pages. Randi gave the okay for us to move forward with making these updates, just so the newest updates are live.

**Main COVID-19 vaccine page** 
**Preview link:** http://preview-prod.vfs.va.gov/preview?nodeId=16359

**Notes:**
- For the last point in the featured content box, the translators added a period. This doesn't match the content on the live English page, but I'm wondering if there is any sort of Spanish grammar rule that would require that period. I removed the period for the time being.


Booster COVID-19 page 
**Preview link:** http://preview-prod.vfs.va.gov/preview?nodeId=34408

**Notes:**
- The note in the section that we're updating had a slightly different format for the text. I went ahead and used the text from the main COVID page which seems to communicate the same information. Feel free to change this if needed, wondering if we'd want to use the language from the Booster page on the main COVID-19 page instead.
